source/usage-examples/watch.txt

```diff
@@ -4,7 +4,53 @@ Watch for Changes
 
 .. default-domain:: mongodb
 
-You can keep track of changes to data in MongoDB, such as changes to a
-collection, database, or deployment, by opening a **change stream**. A change
-stream allows applications to watch for changes to data and react to them.
-The change stream returns **change event** documents when changes occur.
+You can open a change stream on a ``MongoCollection``,
+``MongoDatabase``, or ``MongoClient`` by using the ``Watch()`` method.
+
+The following example passes an aggregation pipeline to the ``Watch()``
+method, which opens a change stream on the ``haikus`` collection and
```

Comment:
```suggestion
The following example uses the ``Watch()``
method to open a change stream on the ``haikus`` collection and
```
Suggestion: I think the bit about aggregation pipelines should be removed. I think aggregation pipelines are an implementation detail that distracts from the action readers came to the page to learn about (how do I watch for changes?).